cli.js

```diff
@@ -1,13 +1,26 @@
 #!/usr/bin/env node
 
 var chalk = require("chalk");
-var lib = require(".");
+var argv = require("yargs").argv;
+var Changelog = require(".").Changelog;
+var ConfigurationError = require(".").ConfigurationError;
 
-var Changelog = lib.Changelog;
-var ConfigurationError = lib.ConfigurationError;
+if (argv.help) {
```

Comment:
Done https://github.com/lerna/lerna-changelog/pull/31/commits/ecb85202af329d93ac4ef9257f88d6d9e59f8303

The _positional arguments_ as far as I can see can only be used for a `command` which in our case is not necessary.